atlas_densities/app/cell_densities.py

```diff
@@ -797,12 +797,15 @@ def fit_average_densities(
     with open(config["realignedSlicesPath"], "r", encoding="utf-8") as file_:
         slices = json.load(file_)
 
+    toLower = "Layer "
     with open(config["cellDensityStandardDeviationsPath"], "r", encoding="utf-8") as file_:
         cell_density_stddev = json.load(file_)
         cell_density_stddev = {
             # Use the AIBS name attribute as key (this is a unique identifier in 1.json)
             # (Ex: former key "|root|Basic cell groups and regions|Cerebrum" -> new key: "Cerebrum")
-            region_name.split("|")[-1]: stddev
+            region_name.split("|")[-1]
```

Comment:
can't we update the input files to have the correct names?

Reply:
Yes but if tomorrow the input file is changed again (by someone else), we may face the same problem. Until we enforce a standard naming when creating the input files, I believe we should have this kind of code protections.

Reply:
However, the transformations that are happening here make it opaque as to what actually needs to be registered in the files - if the contents of the file are expected to be correct, than it's always on the input that the changes need to be made and the code can remain the same.

Reply:
The only transformations happening are `Layer` into `layer` and `/layer` into `, layer`. Are these opaque?

I agree that the content of the files is expected to be correct, but when running the current code one gets
`KeyError: 'Primary motor area, Layer 1'`
and it takes long before figuring out that the region is actually there, just with a lowercase `L`.

Reply:
> I think they're opaque because one has to know the code to know they exist.

Sure, I mean that they are so small that one does not need to know they exist.

Anyways, shall I update the input files or wait for the the `std_cells.json` format to be updated as mentioned [here](https://github.com/BlueBrain/atlas-densities/pull/20#discussion_r1030231804)?

Reply:
> Sure, I mean that they are so small that one does not need to know they exist.

That's true until someone runs across an error caused by things being transfored.  If we don't transform, then things should fail earlier, and people will debug it.

> Anyways, shall I update the input files or wait for the std_cells.json format to be updated as mentioned https://github.com/BlueBrain/atlas-densities/pull/20#discussion_r1030231804?

I think that would be a different PR.

Reply:
@mgeplf, I removed the label transformations. Can you please review it?